backend/app/routers/paybill.py

```
@@ -2,61 +1,199 @@
+        return {"message": "Bill payment already cleared."}
+    is_sandbox = os.getenv("SSLCOMMERZ_IS_SANDBOX", "True").lower() == "true"
+
+    sslcz = SSLCOMMERZ(
+        {
+            "store_id": os.getenv("SSLCOMMERZ_STORE_ID"),
+            "store_pass": os.getenv("SSLCOMMERZ_STORE_PASS"),
+            "issandbox": is_sandbox,
+        }
+    )
+
+    timestamp = datetime.utcnow().strftime("%Y%m%d%H%M%S")
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)